docs/documentation.md

```diff
@@ -1,275 +1,86 @@
-# ˚Annex 3
-
-> KfW's Project Location Data Collection and Management Approach
-
-## Annex 3.1
-
-> The KfW Geodata Model
->
-> In this Section we explain the FC Geodata Model according to which
-> project location data should be collected as well as technical
-> requirements for the data--collection. The goal is to inform KfW
-> staff, PEA, consultants and other external stakeholders responsi- ble
-> for data collection as to the types of location data to be collected
-> and in what form.
->
-> Our FC Geodata Model is based on the IATI standard. The IATI--Standard
-> is used by international development organizations and allows
-> harmonization of FC project loca- tion data across heterogenous
-> projects. The data to be collected are subject to mini- mum
-> requirements in the form of mandatory fields, and there are optional
-> fields flexibly usable for collecting site-specific information.
->
-> Furthermore, projects are able to store single location points (e.g.
-> the location of one power plant site) or a larger number of project
-> sites and features (e.g. 300 locations of buildings in different
-> sectors within the framework of a decentralization project). In
-> addition, project sites with larger areas and more complex geometries
-> (e.g. polygons of forest protection areas) are supported as well.
->
-> Please note that KfW does not publish any exact coordinates of its
-> project locations. Location data collection in fragile and conflict
-> contexts should be treated with extra dil- igence.
-
-#### 3.1.1 General Notes on Geodata and Location Information
+# KfW's Project Location Data Collection and Management Approach
```

Comment:
I compared to the pdf and think that this is a chapter title. Hence, I would argue that we can remove this here.